### PR TITLE
fix comments from CR#3 Yaron

### PR DESCRIPTION
**Pull request by ilanDoron**

### CODE REVIEW

File: contracts/FeeBurner.sol [hunk 1]
```diff
@@ -26,26 +26,25 @@ contract FeeBurner is Withdrawable, FeeBurnerInterface, Utils2 {
...
     KyberNetworkInterface public kyberNetwork;
-    uint public kncPerEthRatePrecision = 600 * 10 ** 18; //--> 1 ether = 600 knc tokens
+    uint public kncPerEthRatePrecision = 600 * PRECISION; //--> 1 ether = 600 knc tokens
 
     function FeeBurner(
         address _admin,
         BurnableToken _kncToken,
         KyberNetworkInterface _kyberNetwork,
-        uint _kncPerEthRatePrecision
+        uint initialRate
```
Comment (yaronvel) [inline contracts/FeeBurner.sol]: initialKncRate or initialKncToEthRate

File: contracts/KyberNetwork.sol [hunk 2]
```diff
@@ -289,7 +290,14 @@ contract KyberNetwork is Withdrawable, Utils2, KyberNetworkInterface, Reentrancy
         returns(uint expectedRate, uint slippageRate)
     {
         require(expectedRateContract != address(0));
-        return expectedRateContract.getExpectedRate(src, dest, srcQty, true);
+        bool includePermissionLess = true;
```
Comment (yaronvel) [inline contracts/KyberNetwork.sol]: Permissionless. Not PermissionLess.

File: contracts/FeeBurner.sol (hunk 1 above)
Comment (yaronvel) [reply contracts/FeeBurner.sol]: also add `_`

File: contracts/KyberNetwork.sol [hunk 3]
```diff
@@ -40,8 +40,9 @@ contract ReentrancyGuard {
 contract KyberNetwork is Withdrawable, Utils2, KyberNetworkInterface, ReentrancyGuard {
 
     enum ReserveType {NONE, PERMISSIONED, PERMISSIONLESS}
-    bytes internal empty;
-    bytes internal permHint = "PERM";
+    bytes internal constant EMPTY_HINT = "";
+    bytes internal constant PERM_HINT = "PERM";
```
Comment (yaronvel) [inline contracts/KyberNetwork.sol]: maybe this should be public? victor could use it.

File: contracts/KyberNetwork.sol [hunk 4]
```diff
@@ -40,8 +40,9 @@ contract ReentrancyGuard {
...
+    uint  public constant PERM_HINT_GET_RATE = 1 << 255; // for get rate. bit mask hint.
```
Comment (yaronvel) [inline contracts/KyberNetwork.sol]: public should probably be before internal.

File: contracts/permissionless/OrderList.sol [hunk 5]
```diff
@@ -183,6 +183,7 @@ contract OrderList is Withdrawable, Utils2, OrderListInterface {
 
     function allocateIds(uint32 howMany) public onlyAdmin returns(uint32) {
         uint32 firstId = nextFreeId;
+        require(nextFreeId + howMany > nextFreeId);
```
Comment (yaronvel) [inline contracts/permissionless/OrderList.sol]: maybe `>=`?

File: contracts/KyberNetwork.sol (hunk 2 above)
Comment (ilanDoron) [reply contracts/KyberNetwork.sol]: fixing
also in event...

File: contracts/KyberNetwork.sol (hunk 3 above)
Comment (ilanDoron) [reply contracts/KyberNetwork.sol]: OK
will do

File: contracts/KyberNetwork.sol (hunk 4 above)
Comment (ilanDoron) [reply contracts/KyberNetwork.sol]: yes
OK
although linter says nonthing.

File: contracts/permissionless/OrderList.sol (hunk 5 above)
Comment (ilanDoron) [reply contracts/permissionless/OrderList.sol]: sure
why not

File: contracts/permissionless/OrderbookReserve.sol [hunk 6]
```diff
@@ -651,15 +656,15 @@ contract OrderbookReserve is OrderIdManager, Utils2, KyberReserveInterface, Orde
...
         uint128 currSrcAmount;
-        address orderMaker;
+        uint32 noUse;
 
         OrderListInterface list = isEthToToken ? ethToTokenList : tokenToEthList;
 
-        (orderMaker, currSrcAmount, currDstAmount,  ,  ) = list.getOrderDetails(orderId);
-        require(orderMaker == maker);
+        (maker, currSrcAmount, currDstAmount, noUse, noUse) = list.getOrderDetails(orderId);
+        require(maker == maker);
```
Comment (yaronvel) [inline contracts/permissionless/OrderbookReserve.sol]: what does that suppose to mean?

**Pull request merged**